### ISSUE TITLE
Spyder help window stops abruptly while scrolling with inertia on Mac

### ISSUE THREAD

**Issue by LudgerHentschel** [spyder-ide/spyder#22587]
### Issue Report Checklist

* [ x] Searched the [issues page](https://github.com/spyder-ide/spyder/issues?q=is%3Aissue) for similar reports
* [ x] Read the relevant sections of the [Spyder Troubleshooting Guide](https://github.com/spyder-ide/spyder/wiki/Troubleshooting-Guide-and-FAQ) and followed its advice
* [ x] Reproduced the issue after updating with ``conda update spyder`` (or ``pip``, if not using Anaconda)
* [ x] Could not reproduce inside ``jupyter qtconsole`` (if console-related)
* [ x] Tried basic troubleshooting (if a bug/error)
    * [x ] Restarted Spyder
    * [ x] Reset preferences with ``spyder --reset``
    * [ x] Reinstalled the latest version of [Anaconda](https://www.anaconda.com/download/)
    * [ x] Tried the other applicable steps from the Troubleshooting Guide
* [x ] Completed the **Problem Description**, **Steps to Reproduce** and **Version** sections below

## Problem Description
The most recent versions of Spyder on my Macs stutter or stall when I scroll the help window/pane "with inertia".  I swipe on the track pad, the text scrolls for a little bit, but then the scrolling stops abruptly. The other windows/panes in Spyder scroll normally. 

Apple refers to this style of scrolling as "with inertia": swipe the trackpad and lift your fingers. When inertia is enabled, the window with continue to sroll for a little bit and then come to a gradual stop. 

### What steps reproduce the problem?

1. Open a help document that is long enough to scroll, for example by typing "np.std(" or "np.mean(" in the console.
2. Move the cursor over the help window and swipe to scroll.
3. The text will scroll but then stop abruptly. 

### What is the expected output? What do you see instead?
Normal scrolling with inertia continues for longer and then comes to a gradual stop. This works fine in the other Spyder windows I use: the editor, the history window, the console. 

### Paste Traceback/Error Below (if applicable)

```python-traceback

PASTE TRACEBACK HERE

```

## Versions

* Spyder version: 6.0.1  (standalone)
* Python version: 3.11.9 64-bit
* Qt version: 5.15.8
* PyQt5 version: 5.15.9
* Operating System: macOS-15.0-x86_64-i386-64bit

This also happened in Spyder 6.0 and under macOS 14.x. It was working fine in Spyder 5.5.6 under macOS 14.x. 

### Dependencies

```
# Mandatory:
aiohttp >=3.9.3                  :  3.10.5 (OK)
asyncssh >=2.14.0,<3.0.0         :  2.17.0 (OK)
atomicwrites >=1.2.0             :  1.4.1 (OK)
chardet >=2.0.0                  :  5.2.0 (OK)
cloudpickle >=0.5.0              :  3.0.0 (OK)
cookiecutter >=1.6.0             :  2.6.0 (OK)
diff_match_patch >=20181111      :  20230430 (OK)
github >=2.3.0                   :  2.4.0 (OK)
importlib_metadata >=4.6.0       :  8.5.0 (OK)
intervaltree >=3.0.2             :  3.1.0 (OK)
IPython >=8.13.0,<9.0.0,!=8.17.1 :  8.27.0 (OK)
jedi >=0.17.2,<0.20.0            :  0.19.1 (OK)
jellyfish >=0.7                  :  1.1.0 (OK)
jsonschema >=3.2.0               :  4.23.0 (OK)
keyring >=17.0.0                 :  25.4.1 (OK)
nbconvert >=4.0                  :  7.16.4 (OK)
numpydoc >=0.6.0                 :  1.8.0 (OK)
parso >=0.7.0,<0.9.0             :  0.8.4 (OK)
pexpect >=4.4.0                  :  4.9.0 (OK)
pickleshare >=0.4                :  0.7.5 (OK)
psutil >=5.3                     :  6.0.0 (OK)
pygments >=2.0                   :  2.18.0 (OK)
pylint >=3.1,<4                  :  3.3.0 (OK)
pylint_venv >=3.0.2              :  3.0.3 (OK)
pyls_spyder >=0.4.0              :  0.4.0 (OK)
pylsp >=1.12.0,<1.13.0           :  1.12.0 (OK)
pylsp_black >=2.0.0,<3.0.0       :  2.0.0 (OK)
pyuca >=1.2                      :  1.2 (OK)
qdarkstyle >=3.2.0,<3.3.0        :  3.2.3 (OK)
qstylizer >=0.2.2                :  0.2.3 (OK)
qtawesome >=1.3.1,<1.4.0         :  1.3.1 (OK)
qtconsole >=5.6.0,<5.7.0         :  5.6.0 (OK)
qtpy >=2.4.0                     :  2.4.1 (OK)
rtree >=0.9.7                    :  1.3.0 (OK)
setuptools >=49.6.0              :  74.1.2 (OK)
sphinx >=0.6.6                   :  8.0.2 (OK)
spyder_kernels >=3.0.0,<3.1.0    :  3.0.0 (OK)
superqt >=0.6.2,<1.0.0           :  0.6.7 (OK)
textdistance >=4.2.0             :  4.6.2 (OK)
three_merge >=0.1.1              :  0.1.1 (OK)
watchdog >=0.10.3                :  5.0.2 (OK)
yarl >=1.9.4                     :  1.9.4 (OK)
zmq >=24.0.0                     :  26.2.0 (OK)

# Optional:
cython >=0.21                    :  3.0.11 (OK)
matplotlib >=3.0.0               :  3.9.2 (OK)
numpy >=1.7                      :  2.1.1 (OK)
pandas >=1.1.1                   :  2.2.3 (OK)
scipy >=0.17.0                   :  1.14.1 (OK)
sympy >=0.7.3                    :  1.13.2 (OK)
```

**Comment by ccordoba12** [comment]
Hey @LudgerHentschel, thanks for reporting. I don't know what we could have changed in Spyder 6 that changed this behavior but we'll try to take a look at it.

@mrclary, could you confirm if this happening for you as well or not? Thanks!

**Comment by mrclary** [comment]
@ccordoba12, yes I've noticed this behavior before. It is not unique to Spyder 6; it has been around a while. I'm not sure what the root cause is, but it may be related to panes that use the webengine.

**Comment by ccordoba12** [comment]
@mrclary, so inertia works well in the Editor and IPython console?

**Comment by LudgerHentschel** [comment]
In my experience scrolling with inertia works as expected in the editor window, history window, files window, and iPython console window. 

Thank you, 

Ludger

**Comment by mrclary** [comment]
> @mrclary, so inertia works well in the Editor and IPython console?

Correct.

> In my experience scrolling with inertia works as expected in the editor window, history window, files window, and iPython console window.

I concur with these observations and also add that inertia works in the Variable Explorer.

**Comment by ccordoba12** [comment]
Ok, thanks for checking that @LudgerHentschel and @mrclary. It's good to know this is not a widespread problem in the interface.

However, I'm going to leave this for 6.2 because it's low priority.

**Comment by LudgerHentschel** [comment]
One more note on this, based on some additional experiments. 

In Spyder's help window, the scroll bar (slider?) is not working properly. In the other windows, the scroll bars seem fine. 

Unfortunately, I don't know that the scroll bar problem is directly related to the scrolling with inertia problem. 

In the help window, the slider starts out much thinner than the sliders in the other windows. 

I can only use the slider if I swipe in the help window and then quickly move the mouse over the slider track. In that case, the slider becomes normal width and I can move it. 

Otherwise, I cannot move the slider. Nor can I click in the slider track to jump around in the window.  

If I swipe in the help window, the slider becomes thin and inactive again. 

If it would be better to open a separate request for this, I can do that. 

Thank you, 

Ludger

**Comment by ccordoba12** [comment]
Thanks for the extra info @LudgerHentschel. You said:

> If it would be better to open a separate request for this, I can do that.

No need to do that. Let's leave both problems in a single place because it seems they are related.
